### PR TITLE
prezi-video

### PR DESCRIPTION
**Pull request by jasmas**
prezi-video: added cask

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew install --cask {{cask_file}}` worked successfully.
- [x] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by jasmas** [comment]
CAVEAT!!!!

The uninstall script pulls up a graphical installer that says install complete and has to be acknowledged. I haven't had a cask uninstall like this before and was not sure how to handle it.

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @jasmas.

Could you please try to add a [`zap` stanza](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md), thanks.

**Comment by jasmas** [comment]
> Thank you for your contribution to `homebrew-cask` @jasmas.
> 
> Could you please try to add a [`zap` stanza](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md), thanks.

done

File: Casks/prezi-video.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            ],
+            delete:    [
+              "/Applications/Prezi Video.app",
+              "/Library/CoreMediaIO/Plug-Ins/DAL/PreziAR.plugin",
+            ]
+
+  zap trash: [
+    "~/Library/Application Support/com.prezi.PreziCast",
+    "/Library/Preferences/com.prezi.PreziCast.plist",
+    "/Library/Preferences/com.prezi.PreziVideo.vcam",
```
Comment (miccal) [inline Casks/prezi-video.rb:26]: Just confirming that these files are in `/Library`, and not  `~/Library`?

Comment (jasmas) [reply Casks/prezi-video.rb]: confirmed

**Comment by miccal** [comment]
Thank you @jasmas.

**Review by miccal — APPROVED** [review]

**Pull request merged**